ucb-rit/coldfront
# Add user registration and require account activation

Fixes #3

**Changes**
- Added a view for registering for an account. This sends an email to the user, prompting him/her to click a confirmation link.
    - Once the link is clicked, the account is activated, and the user may login.
- Provide feedback to inactive users who try to log in.
- Modified `local_strings.py.sample` to use new registration message.

**How to Test**
- Uncomment the email section in `local_settings.py`. Set `EMAIL_BACKEND` to `'django.core.mail.backends.console.EmailBackend'`; emails will be written to the shell. (Re-comment the section when finished to be safe.)
- Copy the updated `local_strings.py.sample` to `local_strings.py`.
- Register a new user. Run through the activation process.
    - An inactive user who attempts to login should be unable to; an activation email should be sent to that user.

**Known Issues**
- Inactive users cannot reset their passwords.
